app/views/includes/foot.pug

```diff
@@ -16,24 +16,30 @@ script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-coo
 
 //Angular UI
 script(type='text/javascript', src='/lib/angular-bootstrap/ui-bootstrap-tpls.js')
-script(type='text/javascript', src='/lib/angular-ui-utils/modules/route.js')
+script(type='text/javascript', src='/lib/angular-ui-utils/modules/route/route.js')
+
+// Every script bundles together
+script(type='text/javascript', src='bundle/main-3cd19faefd.js')
 
 //Application Init
-script(type='text/javascript', src='/js/init.js')
-script(type='text/javascript', src='/js/app.js')
-script(type='text/javascript', src='/js/directives.js')
-script(type='text/javascript', src='/js/filters.js')
+//- script(type='text/javascript', src='/js/init.js')
+//- script(type='text/javascript', src='/js/app.js')
```

Comment:
So, since we are bundling all js files to a single file we don't need the commented out code here.